app/src/main/java/com/example/pix/chat/fragments/MusicRoomFragment.java

```diff
@@ -235,15 +283,30 @@ private void startStream(){
                     remote.getPlayerApi().pause();
                 // Every fixed interval, check for updates to Parse
                 listenerTimer = new Timer();
+                // I need to save this Activity, because I won't be able to call getActivity() if we
+                // change from FriendActivity back to HomeActivity, and our listenerTimer would throw an error
```

Comment:
What does this comment mean?

Reply:
There was actually no need to save the Activity in the variable. I've fixed it.